x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/planner/EsPhysicalOperationProviders.java

```diff
@@ -469,7 +444,7 @@ public SortedSetDocValues ordinals(LeafReaderContext context) {
 
         @Override
         public final String toString() {
-            return "TypeConvertingBlockLoader[delegate=" + delegate + ", convertEvaluator=" + convertEvaluator + "]";
+            return "TypeConvertingBlockLoader[delegate=" + delegate + ", typeConverter=" + typeConverter + "]";
```

Comment:
Perhaps we should implement the TypeConverter.toString method so we control this output?